scripts/run_celery_purge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
```

Comment:
Could we add a warning to not run this against production, with confirmation prompt? 😅

Reply:
Added confirmation:
```
$ scripts/run_celery_purge.sh

--------------------------------------------------

                   WARNING!!!!

    This script is for local development only!

  It will delete everything in the celery queues.

--------------------------------------------------

Are you sure you want to continue?
If so, type 'purge'> 
```